components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -338,6 +286,22 @@ class AutoPushFeature(
         return (System.currentTimeMillis() - prefLastVerified) >= PERIODIC_INTERVAL_MILLISECONDS
     }
 
+    /**
+     * Observers that want to receive updates for new subscriptions and messages.
+     */
+    interface Observer {
+
+        /**
+         * A subscription for the scope is available.
+         */
+        fun onSubscriptionChanged(scope: String) = Unit
```

Comment:
In the README and inline docs above, you have `override fun onSubscriptionChanged(subscription: AutoPushSubscription) { }`, which doesn't match this signature.

Reply:
Thanks, I recently simplified that API and forgot about it. :)